app/javascript/packs/naics-bar-chart-race.js

```diff
@@ -0,0 +1,191 @@
+
+window.addEventListener('DOMContentLoaded', () => {
+  Promise.all([
+    d3.csv('/data/three-digit-sub-sector-01-17.csv', (d) => ({
```

Comment:
Should be able to drop the source file in `/app/assets/data/file.csv` and access it via `/assets/file.csv`. For example try http://127.0.0.1:3000/assets/ma-munis.json when running the server locally. For this to work in production we may need to [disable asset fingerprinting](https://edgeguides.rubyonrails.org/asset_pipeline.html#what-is-fingerprinting-and-why-should-i-care-questionmark) as well. The other option would be to find a csv loader for webpack, but looking further into this it seems like it does some things before we would hit D3 and that is not a rabbit hole I want to jump down anymore.